requre/storage.py

```diff
@@ -86,6 +86,66 @@ def get_base_keys(func: Callable) -> List[Any]:
         return output
 
 
+class StorageKeysInspectOuter(StorageKeysInspect):
+    @staticmethod
+    def get_base_keys(func: Callable) -> List[Any]:
+        """
+        return shorter stack information what will end after info goes to CWD
+        e.g.
+        full stack info for packit project:
+        "long python execute handling":
+          unittest.case:
+            tests_recording.test_status:
+              packit.status:
+                ogr.services.pagure.project:
+                  ogr.services.pagure.service:
+                    requests.sessions:
+                      requre.objects:
+                        requests.sessions:
+                            send:
+        the current one remove all stack info before it goes to current directory (including)
```

Comment:
so this implies that all the code needs to be invoked from $PWD?

Reply:
yep, expecting to run the test commands ideally from root of git, or from tests directory.
This is the weak point of using it. But I were no able to find or find the way, how to met the point where to stop addind stack info, except using PWD, because theoretically it is very dependent on framework what you'll use for test execution and it is exactly reason of current failing, that some change somewhere caused that unittest.test module is not in stack info